lib/acmesmith/post_issueing_hooks/shell.rb

```diff
@@ -0,0 +1,31 @@
+require 'erb'
+require 'acmesmith/post_issueing_hooks/base'
+
+module Acmesmith
+  module PostIssueingHooks
+    class Shell < Base
+      class HostedZoneNotFound < StandardError; end
+      class AmbiguousHostedZones < StandardError; end
+
+      def initialize(common_name:, command:, ignore_failure:false)
+        @common_name = common_name
+        @command = command
+        @ignore_failure = ignore_failure
+      end
+
+      def execute
+        parsed_command = ERB.new(@command).result(binding)
```

Comment:
I request https://github.com/sorah/acmesmith/pull/11#discussion_r90488149 to be implemented for merging this into master branch.